utils/utils.go

```diff
@@ -31,6 +32,11 @@ func GetAPKCTLHomeDir() (string, error) {
 	return os.Getwd()
 }
 
+func GetAPKHelmHomeDir() (string, error) {
+	pwd, err := os.Getwd()
```

Comment:
This needs to be changed. Will have an impact on running integration tests. Let's change this later